demos/blockfactory/cp.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Colour Picker v2.0
+ *
+ * Copyright 2006 Neil Fraser
```

Comment:
Should this be 2019 or did this file come from somewhere else?

Reply:
And same question regarding the lack of jsdoc in this file.

Reply:
The line right below the copyright is the origin:
```
Copyright 2006 Neil Fraser
https://neil.fraser.name/software/colourpicker/
```
This file predates my employment at Google, thus the lack of Google style.